component-lib/src/business/Table/Table.tsx

```diff
@@ -0,0 +1,220 @@
+import React from 'react';
+import cs from 'classnames';
+import _ from 'lodash';
+
+import { Icon } from '../../atoms/Icon';
+import { Checkbox } from '../Checkbox';
+
+type TableBodyCellProps = {
+    rightAligned?: boolean,
+    onClick?: (e?: React.MouseEvent<HTMLTableDataCellElement>) => void
+}
+
+export const TableBodyCell: React.FC<TableBodyCellProps> = (props) => {
+    const { children } = props;
+
+    return (
+        <td onClick={props.onClick ? (e) => {
+                e.stopPropagation();
+                props.onClick && props.onClick();
+            } : undefined}
+            role={props.onClick ? "button" : undefined}
+            onKeyDown={props.onClick ? (e) => {
```

Comment:
We need to always stop propagation here, but only prevent default when there is an onClick. 
```suggestion
            onKeyDown={e => {
        e.stopPropagation();
        if (props.onClick && (e.keyCode === 32 || e.keyCode === 13)) {
          e.preventDefault();
          props.onClick();
        }
      }}
```